### PR TITLE
4749 LRA fixes backport

### LINKED ISSUE #4749: LRA - hardcoded coordinator call timeout
Already fixed in the 3.x and main, backport fix is missing

### PR DESCRIPTION
**Pull request by danielkec**
Fixes #4749

### CODE REVIEW

**Review by Verdent — APPROVED** [review]

File: lra/coordinator/server/src/main/resources/application.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ helidon.lra.coordinator:
     connection:
       url: "jdbc:h2:file:./lra-coordinator"
       username: "h2"
-      password: ""
+      password: ${EMPTY}
```
Comment (tomas-langer) [inline lra/coordinator/server/src/main/resources/application.yaml:31]: This is not correct for version 2.x, we do not understand the ${EMPTY} replacement

File: microprofile/tests/arquillian/src/main/java/io/helidon/microprofile/arquillian/HelidonDeployableContainer.java [hunk 2]
```diff
@@ -422,6 +455,22 @@ public void undeploy(Descriptor descriptor) {
         // No-Op
     }
 
+    /**
+     * Injects the base metric registry and cleans up all metrics in preparation to run another
+     * Arquillian test in the same VM. Without this cleanup, metrics added by a previous test
+     * would be available and may cause failures.
+     */
+    private void cleanupBaseMetrics() {
```
Comment (tomas-langer) [inline microprofile/tests/arquillian/src/main/java/io/helidon/microprofile/arquillian/HelidonDeployableContainer.java:463]: This change seems to be unrelated to LRA. Is it really needed?

**Review by tomas-langer — CHANGES_REQUESTED** [review]

File: microprofile/tests/arquillian/src/main/java/io/helidon/microprofile/arquillian/HelidonDeployableContainer.java (hunk 2 above)
Comment (danielkec) [reply microprofile/tests/arquillian/src/main/java/io/helidon/microprofile/arquillian/HelidonDeployableContainer.java:463]: I have also backported chunk of changes to arqillian deployer to avoid h2 to be generated outside of the target folder, metrics cleanup invocation was part of those changes with aborted deployment cleanup and etc, didn't want to backport just a part of it. 
These changes greatly improve test isolation which is mostly visible only when some tck test fails - only test related to particular change fail, not all the other broken by polluted deployment.

File: lra/coordinator/server/src/main/resources/application.yaml (hunk 1 above)
Comment (danielkec) [reply lra/coordinator/server/src/main/resources/application.yaml]: fixed

**Review by tomas-langer — APPROVED** [review]
